.github/workflows/ci.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         distro:
           - "ubuntu:24.10"
         compiler:
-          - { compiler: GNU13,  CC: gcc-13,   CXX: g++-13,     packages: gcc-13 g++-13 }
+          - { compiler: GNU14,  CC: gcc-14,   CXX: g++-14,     packages: gcc-14 g++-14 }
           - { compiler: LLVM18, CC: clang-18, CXX: clang++-18, packages: clang-18 libomp-18-dev llvm-18-dev libc++-18-dev libc++abi1-18 lld-18 clang-tools-18 mlir-18-tools libmlir-18-dev}
```

Comment:
... but we can't use it until Roman updates the rawspeed submodule (fix is already in develop branch), because currently rawspeed fails to compile by Clang 19. After that happens I'll bump Clang too.